typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -669,6 +682,17 @@ export async function moduleMatchesConfig(
       matches = true;
       break;
     }
+    case IsmType.PAUSABLE: {
+      const pausableIsm = PausableIsm__factory.connect(moduleAddress, provider);
+      const owner = await pausableIsm.owner();
+      matches = matches && eqAddress(owner, config.owner);
```

Comment:
Feel 100% free to ignore this terse but cool syntax but:
`matches &&= eqAddress(owner, config.owner)`